src/components/parameter-tab.tsx

```diff
@@ -0,0 +1,445 @@
+import {
+    getLoadFlowProviders,
+    getLoadFlowSpecificParametersDescription,
+} from 'services/loadflow';
+import {
+    getDefaultLoadFlowProvider,
+    getLoadFlowParameters,
+    getLoadFlowProvider,
+    setLoadFlowParameters,
+    setLoadFlowProvider,
+} from 'services/study/loadflow';
+import { fetchSecurityAnalysisProviders } from 'services/security-analysis';
+import {
+    fetchDefaultSecurityAnalysisProvider,
+    fetchSecurityAnalysisProvider,
+    getSecurityAnalysisParameters,
+    setSecurityAnalysisParameters,
+    updateSecurityAnalysisProvider,
+} from 'services/study/security-analysis';
+import {
+    fetchDefaultSensitivityAnalysisProvider,
+    fetchSensitivityAnalysisProvider,
+    updateSensitivityAnalysisProvider,
+} from 'services/study/sensitivity-analysis';
+import { fetchSensitivityAnalysisProviders } from 'services/sensitivity-analysis';
+import {
+    SensitivityAnalysisParameters,
+    useGetSensitivityAnalysisParameters,
+} from './dialogs/parameters/sensi/sensitivity-analysis-parameters';
+import {
+    ShortCircuitParameters,
+    useGetShortCircuitParameters,
+} from './dialogs/parameters/short-circuit-parameters';
+import {
+    VoltageInitParameters,
+    useGetVoltageInitParameters,
+} from './dialogs/parameters/voltageinit/voltage-init-parameters';
+import {
+    SingleLineDiagramParameters,
+    useGetAvailableComponentLibraries,
+} from './dialogs/parameters/single-line-diagram-parameters';
+import { MapParameters } from './dialogs/parameters/map-parameters';
+import { LoadFlowParameters } from './dialogs/parameters/load-flow-parameters';
+import { SecurityAnalysisParameters } from './dialogs/parameters/security-analysis-parameters';
+import DynamicSimulationParameters from './dialogs/parameters/dynamicsimulation/dynamic-simulation-parameters';
+import { NetworkParameters } from './dialogs/parameters/network-parameters';
+
+export const styles = {
+    title: (theme: Theme) => ({
+        padding: theme.spacing(4),
+    }),
+    panel: (theme: Theme) => ({
+        marginTop: theme.spacing(2),
+        marginBottom: theme.spacing(1),
+    }),
+    tab: {
+        display: 'flex',
+        alignItems: 'center',
+        height: '100%',
+    },
+    listDisplay: (theme: Theme) => ({
+        displat: 'flex',
+        width: '20%',
+        height: '100%',
+        backgroundColor:
+            theme.palette.mode === 'light'
+                ? darken(theme.palette.background.paper, 0.1)
+                : theme.palette.background.paper,
+    }),
+    listItemDisplay: (theme: Theme) => ({
+        paddingLeft: theme.spacing(4),
+    }),
+    parametersBox: (theme: Theme) => ({
+        width: '80%',
+        height: '100%',
+        backgroundColor:
+            theme.palette.mode === 'light'
+                ? theme.palette.background.paper
+                : lighten(theme.palette.background.paper, 0.2),
+        padding: theme.spacing(8),
+    }),
+};
+
+const TAB_VALUES = {
```

Comment:
you can use an [enum](https://www.typescriptlang.org/docs/handbook/enums.html) instead